### PR TITLE
Additional button for show proposal page

### LINKED ISSUE #38: Configurable button for the proposal "show" page.
Repeat the same task as #14 but for creating an additional button after the "back to list" link when visiting a particular proposal.

The main purpose of this is to allow admin to define a "create new proposal" directly when a proposal is created. But with a configurable button we can make it more generic

### PR DESCRIPTION
**Pull request by antopalidi**
fix #38 
screen

### CODE REVIEW

File: app/overrides/decidim/proposals/proposals/show/add_additional_button.html.erb.deface [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<!-- insert_bottom ".m-bottom" -->
+
+<%= render partial: "decidim/proposals/proposals/additional_button_for_show" %>
```
Comment (microstudi) [inline app/overrides/decidim/proposals/proposals/show/add_additional_button.html.erb.deface:3]:
```suggestion
<%= render partial: "decidim/reporting_proposals/proposals/additional_button_for_show" %>
```
We should put this view in our own namespace

**Review by microstudi — CHANGES_REQUESTED** [review]
Also, I'd would like to remove the checkbox:
image

Let's show the button if there's text and link and just not show if there's not.

**Comment by antopalidi** [comment]
> Also, I'd would like to remove the checkbox: image
> 
> Let's show the button if there's text and link and just not show if there's not.

I found it unnecessary to create a checkbox for each custom button. So I left one for both buttons first. Now I added a second checkbox.

**Review by microstudi — APPROVED** [review]
thanks!

**Pull request merged**